src/main/java/uk/ac/cam/cl/dtg/segue/api/GroupsFacade.java

```diff
@@ -471,7 +448,103 @@ public Response deleteGroup(@Context final HttpServletRequest request,
         } catch (NoUserLoggedInException e) {
             return SegueErrorResponse.getNotLoggedInResponse();
         }
-        
+
         return Response.noContent().build();
     }
-}
+
+    @POST
+    @Path("{group_id}/manager")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response addAdditionalManagerToGroup(@Context final HttpServletRequest request,
+                                                @PathParam("group_id") final Long groupId,
+                                                final Map<String, String> responseMap) {
+        if (null == groupId) {
+            return new SegueErrorResponse(Status.BAD_REQUEST, "Group must be specified.").toResponse();
+        }
+
+        if (null == responseMap && responseMap.containsKey("email") && !responseMap.get("email").isEmpty()) {
```

Comment:
Surely `null == responseMap && responseMap.containsKey(...) ....` will lead to a NullPointerException?